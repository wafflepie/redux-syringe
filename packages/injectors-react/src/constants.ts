export const IS_SERVER = Boolean(
	typeof process !== 'undefined' && process && process.versions && process.versions.node
);
