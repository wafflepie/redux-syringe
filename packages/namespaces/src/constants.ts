export const DEFAULT_FEATURE = 'namespaces';
export const NAMESPACE_PREVENTED = 'NAMESPACE_PREVENTED';
