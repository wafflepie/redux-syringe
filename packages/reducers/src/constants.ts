export const ROOT_KEY = '@redux-syringe/reducers/ROOT_KEY';
