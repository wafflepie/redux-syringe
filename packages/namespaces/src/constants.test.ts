import * as constants from './constants';

describe('constants', () => {
	it('should not be changed', () => {
		expect(constants).toMatchSnapshot();
	});
});
