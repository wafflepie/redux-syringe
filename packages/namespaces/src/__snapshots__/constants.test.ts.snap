// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should not be changed 1`] = `
Object {
  "DEFAULT_FEATURE": "namespaces",
  "NAMESPACE_PREVENTED": "NAMESPACE_PREVENTED",
  "__esModule": true,
}
`;
